examples/desktop/app.js

```diff
@@ -49,7 +49,7 @@ app.uiUpdate = function(ui) {
     }
 }
 
-app.loadMapbook({url: 'mapbook.xml'}).then(function() {
+app.loadMapbook({url: 'mapbook-editing.xml'}).then(function() {
```

Comment:
This should be reverted before merge and another example created (or something)

Reply:
Moved to a different mapbook, removed from the default app.js